[Doc Improvement][Meta data fix1] (#12130)
MicrosoftDocs/msteams-docs: msteams-platform/bots/how-to/rate-limit.md

---
title: Rate limiting for bots

> description: Learn how to optimize bot with rate limiting. Detect transient exceptions, per bot thread limit for all bots. You can also perform an exponential backoff.

Reviewer: ```suggestion
description: Learn how to optimize bot with rate limiting, detect transient exceptions, perform an exponential backoff, and know about add per bot thread limit for all bots.
```
